Cargo.toml

```diff
@@ -98,38 +98,38 @@ edition = "2021"
 rust-version = "1.88.0"
 
 [workspace.dependencies]
-spacetimedb = { path = "crates/bindings", version = "1.5.0" }
-spacetimedb-auth = { path = "crates/auth", version = "1.5.0" }
-spacetimedb-bindings-macro = { path = "crates/bindings-macro", version = "1.5.0" }
-spacetimedb-bindings-sys = { path = "crates/bindings-sys", version = "1.5.0" }
-spacetimedb-cli = { path = "crates/cli", version = "1.5.0" }
-spacetimedb-client-api = { path = "crates/client-api", version = "1.5.0" }
-spacetimedb-client-api-messages = { path = "crates/client-api-messages", version = "1.5.0" }
-spacetimedb-codegen = { path = "crates/codegen", version = "1.5.0" }
-spacetimedb-commitlog = { path = "crates/commitlog", version = "1.5.0" }
-spacetimedb-core = { path = "crates/core", version = "1.5.0" }
-spacetimedb-data-structures = { path = "crates/data-structures", version = "1.5.0" }
-spacetimedb-datastore = { path = "crates/datastore", version = "1.5.0" }
-spacetimedb-durability = { path = "crates/durability", version = "1.5.0" }
-spacetimedb-execution = { path = "crates/execution", version = "1.5.0" }
-spacetimedb-expr = { path = "crates/expr", version = "1.5.0" }
-spacetimedb-lib = { path = "crates/lib", default-features = false, version = "1.5.0" }
-spacetimedb-memory-usage = { path = "crates/memory-usage", version = "1.5.0", default-features = false }
-spacetimedb-metrics = { path = "crates/metrics", version = "1.5.0" }
-spacetimedb-paths = { path = "crates/paths", version = "1.5.0" }
-spacetimedb-pg = { path = "crates/pg", version = "1.5.0" }
-spacetimedb-physical-plan = { path = "crates/physical-plan", version = "1.5.0" }
-spacetimedb-primitives = { path = "crates/primitives", version = "1.5.0" }
-spacetimedb-query = { path = "crates/query", version = "1.5.0" }
-spacetimedb-sats = { path = "crates/sats", version = "1.5.0" }
-spacetimedb-schema = { path = "crates/schema", version = "1.5.0" }
-spacetimedb-standalone = { path = "crates/standalone", version = "1.5.0" }
-spacetimedb-sql-parser = { path = "crates/sql-parser", version = "1.5.0" }
-spacetimedb-table = { path = "crates/table", version = "1.5.0" }
-spacetimedb-vm = { path = "crates/vm", version = "1.5.0" }
-spacetimedb-fs-utils = { path = "crates/fs-utils", version = "1.5.0" }
-spacetimedb-snapshot = { path = "crates/snapshot", version = "1.5.0" }
-spacetimedb-subscription = { path = "crates/subscription", version = "1.5.0" }
+spacetimedb = { path = "crates/bindings", version = "1.5.*" }
```

Comment:
Are we certain this doesn't need to be `=1.5.*`?

Reply:
yes, but updated to `=1.5.0` after further discussion